src_erl/Communication_Layer/http_Nerlserver/src/source/sourceStatem.erl

```diff
@@ -89,7 +89,7 @@ idle(cast, {csvList,Workers,CSVData}, State = #source_statem_state{chunkSize = C
   %%  CSVName = lists:last(re:split(CSVPath,"/",[{return,list}])),
   {RouterHost,RouterPort} = getShortPath(MyName,"mainServer",NerlnetGraph),
   %%  send an ACK to mainserver that the CSV file is ready
-  io:format("source updated transmitting list, total batches to send: - ~p~n",[length(CSVlist)]),
+  io:format("source updated transmitting list, available batches to send: - ~p~n",[length(CSVlist)]),
```

Comment:
Try to get use to erlang logger
https://www.erlang.org/doc/man/logger.html

Here you can use info or notice